src/api/routes/accessTokens.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+const { debug, info, warn, error } = require('portal-env').Logger('portal-api:accesstokens');
+const utils = require('./utils');
+const dao = require('../dao/dao');
+
+const accessTokens = require('express').Router();
+
+// ===== SCOPES =====
+
+const ACCESS_TOKENS = 'access_tokens';
+const verifyAccessTokensScope = utils.verifyScope(ACCESS_TOKENS);
+
+// ===== INITIALIZATION =====
+
+const ACCESSTOKEN_CLEANUP_INTERVAL = 600; //seconds (10 minutes)
+
+accessTokens.setup = function (users) {
+    accessTokens._usersModule = users;
+};
+
+accessTokens.initialize = () => {
+    debug('initialize()');
+    if (!dao.isReady()) {
+        debug('auditlog Dao not ready yet: initialize()');
+        setTimeout(accessTokens.initialize, 500);
+        return;
+    }
+    cleanAccessTokens();
+    setInterval(cleanAccessTokens, ACCESSTOKEN_CLEANUP_INTERVAL * 1000);
```

Comment:
"ACCESSTOKEN_CLEANUP_INTERVAL" Isn't this value dependent on the api.settings.token_expiration ?

Reply:
This is just how often the cleanup script runs. What the cleanup routine actually *does* depends on `api.settings.token_expiration` and `api.settings.refresh_token_ttl`. This means that some tokens will linger in the database for some time - but they still won't be valid, as Kong has invalidated them internally.